dts/arm/st/l5/stm32l5.dtsi

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2020 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <arm/armv8-m.dtsi>
+#include <dt-bindings/clock/stm32_clock.h>
+#include <dt-bindings/gpio/gpio.h>
+
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m33";
```

Comment:
@erwango  can you add a node for the number of MPU regions? This is not default:8 for M33, as it is for ARMv7-M.

Reply:
It's 16 indeed, but split into 8 for secure and 8 for non secure.
So, since we define board as non-secure, it has access to 8.